docs/docs/configuration/client.rst

```diff
@@ -65,12 +65,33 @@ Defaults to `true`.
 data-isso-lang
 --------------
 
-Override useragent's preferred language. Isso has been translated in over 12
-languages. The language is configured by its `ISO 639-1
-<https://en.wikipedia.org/wiki/ISO_639-1>`_ (two letter) code.
-
-You find a list of all supported languages on `GitHub
-<https://github.com/posativ/isso/tree/master/isso/js/app/i18n>`_.
+Always render the Isso UI in this language, ignoring what the
+user-agent says is the preferred language.  The default is to
+honor the user-agent's preferred language, and this can be
+specified explicitly by using ``data-isso-lang=""``.
+
+The value of this property should be a `BCP 47 language tag
+<https://tools.ietf.org/html/bcp47>`_, such as "en", "ru", or "pt-BR".
+Language tags are processed case-insensitively, and may use
+underscores as separators instead of dashes (e.g. "pt_br" is treated
+the same as same as "pt-BR").
+
+As of November 2020, Isso has been translated into 25 languages.
```

Comment:
Since this line tends to get out of sync sooner or later maybe just omit it

Reply:
Agreed.